src/publish/cancelled.js

```diff
@@ -0,0 +1,56 @@
+exports.default = async function cancelled({
+  context,
+  github,
+  inputs,
+  Sentry,
+}) {
+  const { repo, version } = inputs;
+
+  const repoInfo = context.repo;
+  const workflowInfo = (
+    await github.rest.actions.getWorkflowRun({
+      ...repoInfo,
+      run_id: context.runId,
+    })
+  ).data;
+  const issue_number = context.payload.issue.number;
+
+  await github.rest.issues.createComment({
+    ...repoInfo,
+    issue_number,
+    body: `Publish workflow cancelled. ([run logs](${
+      workflowInfo.html_url
+    }?check_suite_focus=true#step:8))\n\n_Bad branch? You can [delete with ease](https://github.com/getsentry/${repo}/branches/all?query=${encodeURIComponent(
+      version
+    )}) and start over._`,
+  });
+
+  const release = `${inputs.repo}@${inputs.version}`;
```

Comment:
You already have `repo` and `version` above:
```suggestion
  const release = `${repo}@${version}`;
```